### DISCUSSION TITLE
Automations to/from Home Assistant

### DISCUSSION THREAD

**Discussion by cykeler** [netzero-labs/netzero#27] [Q&A]
Hello, 
Wondering if anyone has tried to implement using the curl commands  from within home assistant? Home assistant has command line sensors that can use curl, or there is a RESTful implementation as well. I'd love to be able to see the Netzero info in there as my whole house is on home assistant.
thanks !

**Comment by zigam** [comment]
I'm not aware of anything, but I think Home Assistant probably has other Tesla Powerwall integration options.

**Reply by cykeler** [reply]
There are a few others out there, like Teslemetry, but I am liking Netzero and am a little concerned about allowing access to too many third parties, but not too sure if that is something I need to worry about. When I saw the automations page and the curl commands it got me wondering.
thanks!
